dp-game-webapp/app/game/play/page.tsx

```diff
@@ -30,60 +34,156 @@ export default function Play() {
-    const percentiles: number[] = simulatedPercentiles(
-      impressions,
-      conversionRate,
-      variance,
-      MAX_COUNT,
-    );
-    setPercentiles(percentiles);
-    setIsLoading(false);
-    preloadInputs();
+  useEffect(() => {
+    const getPercentilesAndPreLoad = () => {
+      const simulatedConversions: Generator<number> =
+        generateSimulatedConversions(
+          impressions,
+          conversionRate,
+          variance,
+          NUM_QUESTIONS,
+        );
+
+      const conversions: number[] = [];
+
+      for (const conversion of simulatedConversions) {
+        conversions.push(conversion);
+        conversions.push(laplaceNoise(conversion, SENSITIVITY, EPSILON));
+      }
+      setConversions(conversions);
+    };
+
+    getPercentilesAndPreLoad();
+  }, []);
+
+  const incrementQuestion = () => {
+    if (currentQuestionIndex < NUM_QUESTIONS * 2 - 1) {
+      setCurrentQuestionIndex(currentQuestionIndex + 1);
+    } else {
+      setIsFinished(true);
+    }
+  };
+
+  const handleDecreaseSpend = () => {
+    incrementQuestion();
+  };
+
+  const handleIncreaseSpend = () => {
+    incrementQuestion();
   };
 
-  getPercentilesAndPreLoad();
-}, []);
-
-const handleDecreaseSpend = () => {
-  // setNextValue();
-};
-
-const handleIncreaseSpend = () => {
-  // setNextValue();
-};
-
-return (
-
-  <div className="mx-auto max-w-7xl px-6 py-32 sm:py-40 lg:px-8">
-    <h1 className="max-w-2xl text-xl font-bold tracking-tight text-gray-900 sm:text-6xl lg:col-span-2 xl:col-auto">
-      <div className="max-w-full p-6 bg-white border border-gray-200 rounded-lg shadow dark:bg-gray-800 dark:border-gray-700">
-        {isLoading ? (
-          <div className="spinner"> Loading...</div>
-        ) : (<>
-          <h5 className="mb-2 text-3xl font-bold tracking-tight text-gray-500 dark:text-white">With these many conversions, would you increase spend or decrease spend?</h5>
-          <p className="mb-2 text-5xl text-grey-700 dark:text-grey-400 justify-end text-center dark:text-white">TODO</p>
-          <div className="flex mt-5 justify-between space-x-3 md:mt-6">
-            <button className="px-3 py-2 text-base font-medium text-white bg-red-700 rounded-lg onClick={handleDecreaseSpend}">
-              Decrease Spend
-            </button>
-            <button className="px-3 py-2 text-base font-medium text-white bg-green-700 rounded-lg onClick={handleIncreaseSpend}">
-              Increase Spend
-            </button>
+  const handleStartButtonClick = () => {
+    setIsStarted(true);
+  };
+  return (
+    <div className="mx-auto max-w-7xl px-6 py-32 sm:py-40 lg:px-8">
+      <section className="py-8">
+        <div className="mt-10 sm:mx-auto sm:w-full sm:max-w-[510px]">
+          <div className="max-w-full p-6 bg-white border border-gray-200 rounded-lg shadow dark:bg-gray-800 dark:border-gray-700">
+            {!isFinished ? (
+              <>
+                {!isStarted ? (
+                  <>
```

Comment:
we should probably throw these into subcomponents soon, even just below in this same file, so that we can understand the logic a bit better.